### PR TITLE
Issue 1 create a started maven project

### PR DESCRIPTION
**Pull request by MistraForest**
@steve2christ Please take a look on this code. I think i supposed to map some members with jpa annotation in Patient to persist them like in case of Address. Give me a feeback. Thx

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+          <package name="" alias="false" withSubpackages="true" />
+          <package name="java" alias="false" withSubpackages="true" />
+          <package name="javax" alias="false" withSubpackages="true" />
+          <package name="kotlin" alias="false" withSubpackages="true" />
+          <package name="" alias="true" withSubpackages="true" />
+        </value>
+      </option>
+    </JetCodeStyleSettings>
+  </code_scheme>
+</component>
```
Comment (steve2christ) [inline .idea/codeStyles/Project.xml:22]: Please delete diese ordner

File: .idea/codeStyles/codeStyleConfig.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
+  <state>
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
+  </state>
+</component>
```
Comment (steve2christ) [inline .idea/codeStyles/codeStyleConfig.xml:5]: same comment like before

File: .idea/compiler.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    </annotationProcessing>
+    <bytecodeTargetLevel>
+      <module name="si-projekt-anamnese-service" target="1.8" />
+    </bytecodeTargetLevel>
+  </component>
+  <component name="JavacSettings">
+    <option name="ADDITIONAL_OPTIONS_OVERRIDE">
+      <module name="si-projekt-anamnese-service" options="-parameters" />
+    </option>
+  </component>
```
Comment (steve2christ) [inline .idea/compiler.xml:21]: same comment like before

File: .idea/encodings.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="Encoding">
+    <file url="file://$PROJECT_DIR$/src/main/java" charset="UTF-8" />
+  </component>
+</project>
```
Comment (steve2christ) [inline .idea/encodings.xml:6]: same comment like before

File: .idea/jarRepositories.xml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+      <option name="name" value="Maven Central repository" />
+      <option name="url" value="https://repo1.maven.org/maven2" />
+    </remote-repository>
+    <remote-repository>
+      <option name="id" value="jboss.community" />
+      <option name="name" value="JBoss Community repository" />
+      <option name="url" value="https://repository.jboss.org/nexus/content/repositories/public/" />
+    </remote-repository>
+  </component>
+</project>
```
Comment (steve2christ) [inline .idea/jarRepositories.xml:20]: same comment like before

File: .mvn/wrapper/MavenWrapperDownloader.java [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+ * limitations under the License.
+ */
+import java.net.*;
+import java.io.*;
+import java.nio.channels.*;
+import java.util.Properties;
+
+public class MavenWrapperDownloader {
+
+    private static final String WRAPPER_VERSION = "0.5.6";
```
Comment (steve2christ) [inline .mvn/wrapper/MavenWrapperDownloader.java:23]: same comment like with the ordner .idea for the ordner .mvn

File: si-projekt-anamnese-service.iml [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+        <naming-strategy-map />
+      </configuration>
+    </facet>
+    <facet type="Spring" name="Spring">
+      <configuration />
+    </facet>
+    <facet type="web" name="Web">
+      <configuration>
+        <webroots />
+        <sourceRoots>
```
Comment (steve2christ) [inline si-projekt-anamnese-service.iml:21]: Please delete this file befor pushing

File: src/main/java/thb/siprojektanamneseservice/model/Diagnosis.java [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+public class Diagnosis {
+
+    @Id
+    @GeneratedValue(generator = "uuid2")
+    @GenericGenerator(name = "uuid2", strategy = "uuid2")
+    private UUID id;
+
+    private String date;
+    private VegetativeAnamnesisDecision type;
+    private List<TypeOfExamination> typeOfExaminations;
```
Comment (steve2christ) [inline src/main/java/thb/siprojektanamneseservice/model/Diagnosis.java:27]: Please change **typeOfExaminations** to **examinations**

File: src/main/java/thb/siprojektanamneseservice/model/FamilyAnamnesis.java [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+@Entity
+public class FamilyAnamnesis {
+
+    @Id
+    @GeneratedValue(generator = "uuid2")
+    @GenericGenerator(name = "uuid2", strategy = "uuid2")
+    private UUID id;
+
+    private PreExistingIllness father;
+    private PreExistingIllness mother;
```
Comment (steve2christ) [inline src/main/java/thb/siprojektanamneseservice/model/FamilyAnamnesis.java:25]: **father** and **mother** shall be a list of **PreExistingIllness**

File: src/main/java/thb/siprojektanamneseservice/model/AllergyType.java [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+    FRUCHTZUCKER("fructose"),
+    NAHRUNGSMITTEL("foods"),
+    POLLEN("pollen"),
+    ROENTGENKONTRASTMITTEL("radio_opaque_substance"),
+    TIERHAAR("animal_hair"),
+    LOKALANAESTHETIKA("local_anaesthetics"),
+    MILCHZUCKER("lactose"),
+    HAUSSTAUB("House_dust"),
+    KONSERVIERUNGSSTOFFE("antidegradant"),
+    VITAMIN_B("vitamin_b");
```
Comment (steve2christ) [inline src/main/java/thb/siprojektanamneseservice/model/AllergyType.java:15]: Please also translate the attributes names to english (FRUCHTZUCKER, NAHRUNGSMITTEL,...)

File: src/main/java/thb/siprojektanamneseservice/model/MaritalStatus.java [hunk 11]
```diff
@@ -0,0 +1,15 @@
+package thb.siprojektanamneseservice.model;
+
+public enum MaritalStatus {
+    SINGLE("single"),
+    PARTNERSCHAFT("civil_union"),
+    VERHEIRATET("married"),
+    GESCHIEDEN("divorced"),
+    VERWITWET("widowed");
```
Comment (steve2christ) [inline src/main/java/thb/siprojektanamneseservice/model/MaritalStatus.java:8]: Please also translate the attributes names to english (PARTNERSCHAFT, VERHEIRATET,...)

File: src/main/java/thb/siprojektanamneseservice/model/MedicationIntake.java [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+public class MedicationIntake {
+
+    @Id
+    @GeneratedValue(generator = "uuid2")
+    @GenericGenerator(name = "uuid2", strategy = "uuid2")
+    private UUID id;
+
+    private String designation;
+    private String dosage;
+    private String startDate;
```
Comment (steve2christ) [inline src/main/java/thb/siprojektanamneseservice/model/MedicationIntake.java:26]: Please change the type from String tu Date (java.util)

File: src/main/java/thb/siprojektanamneseservice/model/VegetativeAnamnesisDecision.java [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+
+public enum VegetativeAnamnesisDecision {
+
+    BRENNEN("burning"),
+    DURCHFALL("diarrhea"),
+    DURCHFALL_MIT_BLUT("diarrhea_With_blood"),
+    NORMAL("normal"),
+    OEFTER("frequently"),
+    START_SCHWIERIGKEITEN("teething_troubles"),
+    VERMEHRT("increased");
```
Comment (steve2christ) [inline src/main/java/thb/siprojektanamneseservice/model/VegetativeAnamnesisDecision.java:11]: Please also translate the attributes names to english (BRENNEN, DURCHFALL,...)

**Comment by steve2christ** [comment]
Please squatch all commits. there is actually 23 commits for just one pull

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (MistraForest) [reply .idea/codeStyles/Project.xml:22]: Done!

File: src/main/java/thb/siprojektanamneseservice/model/Diagnosis.java (hunk 8 above)
Comment (MistraForest) [reply src/main/java/thb/siprojektanamneseservice/model/Diagnosis.java:27]: done

File: src/main/java/thb/siprojektanamneseservice/model/FamilyAnamnesis.java (hunk 9 above)
Comment (MistraForest) [reply src/main/java/thb/siprojektanamneseservice/model/FamilyAnamnesis.java:25]: done

**Comment by MistraForest** [comment]
issue fixed

**Comment by MistraForest** [comment]
Hey @steve2christ  I make the changes.
Can i merge or you do it?

**Comment by MistraForest** [comment]
@steve2christ 
I think it ok now, I'll merge it!

**Pull request merged**